### PR TITLE
feat(ui-framework): Update to the latest version of ui-framework

### PR DESCRIPTION
**Pull request by pavlovicnemanja**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
     "sass-loader": "6.0.6",
     "shortid": "^2.2.8",
     "style-loader": "0.18.2",
-    "superdesk-ui-framework": "superdesk/superdesk-ui-framework#195e5ca",
+    "superdesk-ui-framework": "superdesk/superdesk-ui-framework#957eac552",
```
Comment (petrjasek) [inline package.json]: maybe it's time to release some 1.0 version, what do you think?

Comment (pavlovicnemanja) [reply package.json]: we have it, v1.0, it's final version before refactoring to webpack and adding react. After I fix all issues, I will create new release and add it :)

Comment (petrjasek) [reply package.json]: do you push this to npm? it should be set then like `"superdesk-ui-framework": "1.0.0",`

Comment (pavlovicnemanja) [reply package.json]: still not, but I will make that also! :)

**Pull request closed without merging**